### PR TITLE
refactor: use fs' built in promise support

### PR DESCRIPTION
**Pull request by ChristianMurphy**
https://nodejs.org/api/fs.html#fs_filehandle_readfile_options

### CODE REVIEW

**Review by BarryThePenguin — APPROVED** [review]
Only thing I'd consider changing is moving `.toString()` to each usage of `content`

Otherwise looks good 👀

**Comment by ChristianMurphy** [comment]
> Only thing I'd consider changing is moving `.toString()` to each usage of `content`

Good idea, refactored in 91562b5

**Review by wooorm — APPROVED** [review]

File: src/utils/workspace.ts [hunk 1]
```diff
@@ -26,14 +26,14 @@ export async function parseWorkspaceConfig(files : vscode.Uri[]) {
 		}
 	}
 
-	const content = await fileRead(files[0].fsPath);
+	const content = await fs.readFile(files[0].fsPath);
```
Comment (asbjornu) [inline src/utils/workspace.ts:29]: With this replaced, I think we can delete the entire `fs.ts` file, no?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: src/utils/workspace.ts (hunk 1 above)
Comment (ChristianMurphy) [reply src/utils/workspace.ts:29]: yes, and it is removed as part of this PR already.

Comment (asbjornu) [reply src/utils/workspace.ts:29]: Sorry, I didn't notice. 🙈

**Review by asbjornu — APPROVED** [review]

**Pull request merged**